host/src/c_api/c_api.cc

```diff
@@ -1289,6 +1289,12 @@ XGB_DLL int add_client_key(uint8_t* data, size_t data_len, uint8_t* signature, s
   safe_ecall(enclave_add_client_key(Enclave::getInstance().getEnclave(), &Enclave::getInstance().enclave_ret, data, data_len, signature, sig_len));
 }
 
+
+XGB_DLL int sync_client_key() {
```

Comment:
See comment for `c_api.h`. Not sure if we need to expose this function to the client.